tests/fragments/google/api/httpbody.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 Google LLC
```

Comment:
Date!

Reply:
This file was copied from https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto which is why I kept the date. It's not clear if we should update the date when the file is copied from another repository.